tests/test_edgeql_functions.py

```diff
@@ -2083,7 +2083,7 @@ async def test_edgeql_functions_to_str_07(self):
                 await self.con.fetchall(
                     r'''SELECT to_str(<cal::local_time>'15:01:22', '');''',)
 
-    async def test_edgeql_functions_to_str_08(self):
+    async def test_edgeql_functions_array_join_01(self):
         await self.assert_query_result(
             r'''SELECT to_str(['one', 'two', 'three'], ', ');''',
```

Comment:
`SELECT to_str(['one', 'two', 'three'], ', ');` needs to be changed to `SELECT array_join(['one', 'two', 'three'], ', ');`

Reply:
Sorry. I did not check this. I will do it for both functions.